v0/destinations/marketo/util.js

```diff
@@ -39,6 +40,7 @@ const marketoApplicationErrorHandler = (
   const { response } = marketoResponse;
   const { errors } = response;
   if (errors && MARKETO_ABORTABLE_CODES.indexOf(errors[0].code) > -1) {
+    console.log("in abortable");
```

Comment:
No console statements. Use logger.

Reply:
Don't think we'd need logger here. Removed them